packages/components/src/components/tab/tab.ts

```diff
@@ -74,7 +74,7 @@ export default class SdTab extends SolidElement {
       <div
```

Comment:
@Vahid1919 If I look at the diff in chromatic, it already changes the size of the tab even though you did not change the length of the lable.

Reply:
@karlbaumhauer yes, that's because I set an explicit width for the tab before. I now realised that does not make sense since the width of the tab should be determined by the width of the text inside. This new width is based on the size of the text alone.